WordPressSharedTests/NSStringSummaryTests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+@testable import WordPressShared
+
+class NSStringSummaryTests: XCTestCase {
+
+    func testSummaryForContent() {
+        let content = "<p>Lorem ipsum dolor sit amet, [shortcode param=\"value\"]consectetur[/shortcode] adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p> <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p><p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p><p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>"
+        let expectedSummary = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, …"
+        
+        let summary = content.summarized()
+        
+        XCTAssertEqual(summary, expectedSummary)
```

Comment:
What do you think about moving `expectedSummary` to here? It seems that this is part of the _assert_ (not the _arrange_).

(The same for the other unit tests)

Reply:
As mentioned in our chat - I tend to prefer the expectations to be clear on top.  The reason is that I feel that documents the purpose of the entire test more clearly.

Since we agreed to disagree (😆) over DM, I'm marking this as resolved.